design/cli-install-changes.md

```diff
@@ -0,0 +1,156 @@
+   
+2) `velero set`
+All the other configuration that is not component specific nor necessary for initialization. Mostly things to be updated. Might be grouped under `init`.
+
+```
+      --pod-annotations mapStringString            annotations to add to the Velero and restic pods. Optional. Format is key1=value1,key2=value2
+      --restore-only                               run the server in restore-only mode. Optional.
+      --pod-cpu-limit string                       CPU limit for Velero pod. A value of "0" is treated as unbounded. Optional. (default "1000m")
+      --pod-cpu-request string                     CPU request for Velero pod. A value of "0" is treated as unbounded. Optional. (default "500m")
+      --pod-mem-limit string                       memory limit for Velero pod. A value of "0" is treated as unbounded. Optional. (default "256Mi")
+      --pod-mem-request string                     memory request for Velero pod. A value of "0" is treated as unbounded. Optional. (default "128Mi")
+```
+
+3) `velero config`
+Component specific configuration for both backup and snapshot locations. 
+
+Much like `kubectl config` is about clusters and context, so does this `velero config` should be only about the backup/snapshot locations.
+
+```
+      --no-secret                                       flag indicating if a secret should be created. Must be used as confirmation if --secret-file is not provided. Optional.
+      --secret-file string                              file containing credentials for backup and volume provider. If not specified, --no-secret must be used for confirmation. Optional.
+
+      --backup-location mapStringString                 configuration to use for creating a backup storage location. Format is key1=value1,key2=value2 (was `backup-location-config`)
+      --get-backup-locations                            Display backup storage locations
+      --current-backup-location                         displays the current default backup storage location (NEW)
+      --use-backup-location string                      sets the default backup storage location (default "default") (was `default-backup-storage-location`)
+
+      --snapshot-location  mapStringString              configuration to use for creating a volume snapshot location. Format is key1=value1,key2=value2 (was `snapshot-location-config`)
+      --get-snapshot-locations                          Display snapshot locations
+      --current-snapshot-locations                      displays the current default volume snapshot locations (NEW)
+      --use-snapshot-locations mapStringString          sets the list of unique volume providers and default volume snapshot location (provider1:location-01,provider2:location-02,...) (was `default-volume-snapshot-locations`)
+
+      --set-default-location                            configuration to create a default locations. Format is bucket=value,prefix=value,plugin-name=value,snapshot-location=true/false. Optional.
```

Comment:
`location` may be be overloaded. Can it be called something else?
Also is this the default `backup-location` or `snapshot-location`

Reply:
Just FYI, this is going to be removed.